packages/common/src/lib/list/list.component.scss

```diff
@@ -40,6 +40,16 @@ mat-list {
   height: $igo-list-item-height;
 }
 
+:host >>> .mat-list .mat-list-item .mat-list-text .igo-catalog-group-title {
```

Comment:
Catalog and search stuff should be their own modules. Nothing in the common packages should be aware of their existence.